SCloudRandomizer/MusicSource.h

```diff
@@ -17,7 +17,7 @@ typedef void(^tracksFetchedCompletionHandler)(NSArray* tracks);
 typedef void(^singleTrackFetchedCompletionHandler)(Track* track);
 
 + (MusicSource *) getInstance;
-- (BOOL)isUserLoggedIn;
+- (SCAccount *)isUserLoggedIn;
```

Comment:
Dont need to change this to return BOOL because we can check we can check the returned object jsut like a null. ie, if (SCAccount) which is the same as if (isUserLoggedIn).